Merge www and master branches, clean up repo & update safari compatibility

idaknow opened the pull request:
As per previous pull requests #24 #25 this is the updated pull request to merge WWW and MASTER branch as per issue #81 on the BrownHCI repository.

* The www directory is the website, and can be copied across without having to run any scripts beforehand (grunt command copies across the webgazer.js to the www repository & there is a seperate node module for both the website and the build).
* Many files have been moved around in attempt to clean up the repository and make it more organised.
* The website pages are updated in regards to Safari compatibility

Potential issues detected: I am working from a mac environment and it appears there are a lot of changes to the binary of files, so it would be good to check it works in other development environments, such as windows.

Libby713 commented:
Overall comments:

- build/out/font only contains OpenSans-Bold-webfont.eot, should OpenSans-Bold-webfont.svg and OpenSans-Bold-webfont.woff also be present like with fonts elsewhere?
- ridgeWorker.js.html missing from www/documentation/
- Where did demo.mp4 get moved to? It isn’t in www/media/video
- Should LICENCE.md go into LICENCE/ too?

idaknow commented:
As per the comments by @Libby713 

- Updated .gitignore: doesn't include package-lock.json and excludes demo.m4 (now demo.mp4 is added to the repo)
- Updated capitalisation in README
- Updated build/out/ directory with the same copy from master
- Added www/documentation/ridgeWorker.js.html

As per slack, moved all the license files back to the root directory

I think that's everything resolved, thus far.

Jack-Wong94 commented on README.md:
```diff
@@ -34,24 +34,31 @@ To use the webgazer script in the head of an HTML file add the `async` tag to en
+	cd www
+	npm install
+	# Run the webpage index.html as a server
+	browser-sync start --server --files "*"
```
Because the index html is under the www folder, this command line won't really work. Instead it should be changed to `browser-sync start --server --files --startPath "www/index.html"

idaknow replied:
Does this not work for you? It works for me if you are in the www folder. Alex will copy across only the www folder onto the server.

Jack-Wong94 replied:
No it does not work for me if i use the original line if I am in the www folder. I have encountered this problem when I worked on this merge branch before. For some reason, browsersync cannot reference to the file in the parent directory. To solve this problem, I use the new command line when I am in the parent directory and it works

idaknow replied:
But it should not be accessing anything in the parent directory anymore. Have you followed the new instructions?

Libby713 commented:
I've run the branch and it all works fine for me :)

Libby713 commented on www/index.html:
```diff
@@ -0,0 +1,451 @@
+                    <li><a href="http://cs.brown.edu/~nediyana">Nediyana Daskalova</a> - PhD student at Brown University</li>
+                    <li><a href="http://www.cc.gatech.edu/~hays/">James Hays</a> - Associate Professor at Georgia Institute of Technology</li>
+                    <li><a href="https://www.linkedin.com/in/idakdesmet/">Ida De Smet</a> - Software engineering student at the University of Auckland</li>
+                    <li>Elizabeth Stevenson - Software engineering student at the University of Auckland</li>
```
@idaknow  would you mind adding the link to my linkedin account here too? if that's not a hassle!
https://www.linkedin.com/in/elizabeth-stevenson-9b47a7125/

Libby713 replied:
@Jack-Wong94 do you have a linkedin account?

jeffhuang commented:
thank you all!!
